README.md

```diff
@@ -40,6 +49,36 @@ Run the generator from within the new folder:
 $ cd my-storefront && slush storefront
 ```
 
+### Generating StoreFront Components
+
+#### via Prompts
+
+To generate a component by answering a series of questions, execute the following:
+
+```
+slush storefront:component
+```
+
+#### via CLI
+
+To generate a component directly from the command line, execute the following:
+
+```
+slush storefront:component <ComponentName> <path/to/components/directory>
+```
+
+When generating the component's definition and test files, the path argument is implicitly relative to the project's source code and test code directories (by default, these are assumed to be `src/` and `test/unit`). The following flags may be used to override these defaults:
+
+```
+// Include the --srcRoot flag to overwrite the default source code directory.
```

Comment:
Don't use `//` as comments in shell. Use `#`.

Reply:
Good call, will update.

Reply:
Implemented, resolving.